modules/registry-common-terms.adoc

```diff
@@ -9,10 +9,15 @@
 This glossary defines the common terms that are used in the registry content.
 
 container::
-Lightweight and executable images that consist software and all its dependencies. Because containers virtualize the operating system, you can run containers in data center, a public or private cloud, or your local host.
+Lightweight and executable images that consist of software and all its dependencies. Because containers virtualize the operating system, you can run containers in a data center, a public or private cloud, or your local host.
 
 Image Registry Operator::
+ifdef::openshift-rosa-hcp[]
+The Image Registry Operator runs in the `HOSTED_CLUSTERS_NAMESPACE` of the management cluster, and manages the registry instance in openshift-image-registry of the cluster.
```

Comment:
`HOSTED_CLUSTERS_NAMESPACE` need replace to `CONTROL_PLANE_NAMESPACE` to keep consistent with https://docs.openshift.com/container-platform/4.16/hosted_control_planes/hcp_high_availability/hcp-backup-restore-on-premise.html#hosted-cluster-etcd-backup-restore-on-premise_hcp-backup-restore-on-premise 

In this file, HOSTED_CLUSTERS_NAMESPACE is `clusters` namespeace

Reply:
Done thanks